File: isem-finanzas-web/src/app/@finanzas/@auditorias/models/oficio.model.ts
export class OficioModel {
  id_oficio?: number;
  id_auditoria?: number;
  num_oficio?: string;
  fecha_recepcion?: string;
  fecha_cumplimiento?: string;
  id_area_remitente?: number;
  id_persona_remitente?: number;
  descripcion?: string;
  activo?: boolean;
  folio_auditoria?: string;
  nombre_auditoria?: string;
  nombre_area_remitente?: string;
  nombre_persona_remitente?: string;
  id_usuario_oficio?: number;
  nombre_usuario_oficio?: string;
  id_estatus?: number;
  nombre_estatus?: string;
  solicitar_prorroga?: boolean;
  id_usuario_busqueda?: number;
}

File: isem-finanzas/common/backend/database/Models/Modelos/OficioModel.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.ComponentModel.DataAnnotations.Schema;

namespace Models.Modelos
{
    public class OficioModel
    {
        public int? id_oficio { get; set; }
        public int? id_auditoria { get; set; }
        public string num_oficio { get; set; }
        public string fecha_recepcion { get; set; }
        public string fecha_cumplimiento { get; set; }
        public int? id_area_remitente { get; set; }
        public int? id_persona_remitente { get; set; }
        public string descripcion { get; set; }
        public bool? activo { get; set; }
        public string folio_auditoria { get; set; }
        public string nombre_auditoria { get; set; }
        public string nombre_area_remitente { get; set; }
        public string nombre_persona_remitente { get; set; }
        public int? id_estatus { get; set; }
        public string nombre_estatus { get; set; }
        public int? id_usuario_oficio { get; set; }
        public string nombre_usuario_oficio { get; set; }
        [NotMapped] public bool? solicitar_prorroga { get; set; }
        [NotMapped] public int? id_usuario_busqueda { get; set; }
        [NotMapped] public int tipoOperacion { get; set; }
    }
}

